builder/dockerfile/builder.go

```diff
@@ -215,6 +215,9 @@ func (b *Builder) build(config *types.ImageBuildOptions, context builder.Context
 
 	var shortImgID string
 	for i, n := range b.dockerfile.Children {
+		if i == len(b.dockerfile.Children)-1 {
```

Comment:
can we have a comment here, so we remember we only want to add labels to the last layer?